cdbsearch.py

```diff
@@ -294,7 +334,7 @@ async def search(self, board, depth):
         ):
             return 0, ["draw"]
 
-        # get current ranking, use an executor to limit total requests in flight
+        # get current ranking, use an executor to limit total requests in flight TODO: update this comment
```

Comment:
@vondele What would be a good comment here?

Reply:
just `# get current ranking` the other part is commented elsewhere (and currently not really right).